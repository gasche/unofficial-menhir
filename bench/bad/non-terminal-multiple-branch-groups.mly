%start<int> a
%%

b: {}
b: {}
