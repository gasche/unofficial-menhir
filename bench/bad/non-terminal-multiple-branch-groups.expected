File "non-terminal-multiple-branch-groups.mly", line 4, characters 0-1:
File "non-terminal-multiple-branch-groups.mly", line 5, characters 0-1:
Error: the nonterminal symbol b is multiply defined.
